site.config.js

```diff
@@ -17,9 +17,9 @@ module.exports = {
     logoSrc: 'https://github.com/cerner/terra-core/raw/master/terra.png',
 
     /* The title for the site header. */
-    title: 'Terra',
+    title: 'Terra Framework',
 
-    /* The subtitle for the site header. */
-    subtitle: 'Framework',
+    /* tempary fix to allow for bidirecitonality in site. Current v2.0.0-RC.0 release has wrong default value. */
```

Comment:
I can release a 2.0.0-RC.1 if you want, then we don't need to make temporary fixes.

Reply:
Sounds good to me.